client/src/utils/mutations.js

```diff
@@ -54,4 +54,18 @@ mutation removeBook($bookId: String!) {
         }
     }
 }
-`
\ No newline at end of file
+`
+export const UPDATE_BOOK_STATUS = gql`
+mutation updateBookStatus($bookId: ID!, $isRead: Boolean!, $isReading: Boolean!) {
+  updateBookStatus(bookId: $bookId, isRead: $isRead, isReading: $isReading) {
+    _id
+    title
+    author
+    pages
+    genre
```

Comment:
We don't have a genre coming from the API. We aren't using genre in our books Schema.